### PR TITLE
Initial work on client commands

### PR DESCRIPTION
**Pull request by AlexanderDzhoganov**
This is a very basic implementation of a way to send commands from the client but it seems to work. 

Also I've included a way to send commands for debugging in [this commit](https://github.com/AlexanderDzhoganov/pipboyclient/commit/687b863b4a7530584aa66f39141a9ca45cf8f266) to my client UI app.

### CODE REVIEW

File: lib/pipdecode.js [hunk 1]
```diff
@@ -34,10 +34,11 @@ module.exports = function() {
     }
 
     if(this.buffer.length >= this.expectedSize) {
-      if(size !== this.expectedSize) {
-        this.emit('error', 'Unexpected size change')
-      }
       var size = this.buffer.readUInt32LE(0);
+      if(size !== this.expectedSize - 5) {
+        this.emit('error', 'Unexpected size change');
+      }
```
Comment (rgbkrk) [inline lib/pipdecode.js]: D'oh! Did I seriously introduce this? I'm sorry. I'll bring this part of the patch in now.

Comment (rgbkrk) [inline lib/pipdecode.js]: :wrench: fixed this

**Comment by rgbkrk** [comment]
I think me merging your other PR @AlexanderDzhoganov ended up with another merge conflict you'll have to do an interactive rebase on.

**Comment by AlexanderDzhoganov** [comment]
Conflicts should be resolved now.

**Comment by rgbkrk** [comment]
Looking good. If in the future you want a clean commit history, [Atlassian has a great tutorial on rebase](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase-i). Depending on your editor, there are great tools for assisting in the interactive rebase too.

**Comment by AlexanderDzhoganov** [comment]
Clean commit history sounds nice, I've squashed the commits on all PRs.

**Pull request merged**